llvm/include/llvm/Transforms/VoronovaLab2/VoronovaLab2.h

```diff
@@ -1,5 +1,8 @@
 #pragma once
 
+#define DEBUG_TYPE "VoronovaLab2"
```

Comment:
@VoronovaAnastasia, this macro might generate conflicts please move it into appropriate place